service/src/main/resources/db/migration/V23__mobile_device.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE mobile_device(
+    id uuid PRIMARY KEY REFERENCES employee(id),
+    created timestamp with time zone DEFAULT now() NOT NULL,
+    updated timestamp with time zone DEFAULT now() NOT NULL,
+    unit_id uuid NOT NULL REFERENCES daycare(id),
+    name text NOT NULL,
+    deleted boolean NOT NULL DEFAULT false
+);
+
```

Comment:
Let's also add

`CREATE INDEX idx$mobile_device_unit ON mobile_device (unit_id) WHERE deleted IS FALSE`

as a sensible default